### PR TITLE
Add option to send arguments to pytest when used through tox

### PR DESCRIPTION
**Pull request by lucifurtun**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -6,7 +6,7 @@ ignore = E129,W504
 changedir = tests
 deps = pytest
        requests
-commands = pytest
+commands = pytest -v {posargs} -p aiohttp.pytest_plugin -p colibris.test
```
Comment (ccrisan) [inline tox.ini]: Why `-p colibris.test`? That's a package intended for the Colibris-based project tests, not for Colibris itself.

Comment (lucifurtun) [reply tox.ini]: Not sure about it, don't we need it? But I'm sure we need `aiohttp.pytest_plugin`.

Comment (lucifurtun) [reply tox.ini]: Removed!

**Comment by lucifurtun** [comment]
@ccrisan, ready to merge?

**Pull request merged**